AUBGTheHUB/monolith
# #425 Swap custom commit hooks for husky

Add husky and lint-staged for better control over what files are being listed on commit. 
Python pre-commit will be added in the future as well.

## How to verify:
Run the `make install-hooks` phony.
Go to a js file, add some minor change and try to commit it. File should be formatted following the new guidelines found in `prettierrc`.
